src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -429,11 +429,7 @@ public int compareOffsets(final NakadiCursor first, final NakadiCursor second) t
 
     @Override
     public void validateCommitCursor(final NakadiCursor position) throws InvalidCursorException {
```

Comment:
Yes, true. There's no need to call validateCursorForNulls here. I removed it now.
 I call KafkaCursor.fromNakadiCursor for validation purposes (it will throw InvalidCursorException if something is wrong), so I don't really need Kafka cursors that it returns.